### DISCUSSION TITLE
I am trying to map my persistable package using morphiaConfig.packages(list). But it is not able detect it.

### DISCUSSION THREAD

**Discussion by SOCRATESZZ** [MorphiaOrg/morphia#3160] [General]
I want morphia to stop detecting all packages in project and only scan the persistable package. 
```
MorphiaConfig morphiaConfig = MorphiaConfig.load();
        morphiaConfig.applyIndexes(true);
        morphiaConfig.applyDocumentValidations(this.applyDocumentValidation);
        morphiaConfig.autoImportModels(this.autoImportModels);
        morphiaConfig.storeEmpties(true);
        morphiaConfig.database(this.databaseName);
        List<String> list = new ArrayList();
        list.add(this.packages);
        morphiaConfig = morphiaConfig.packages(list);
        return morphiaConfig;
```
I cannot pass the morphia.config.properties file. since I compile it in a library and use it in a different project.  I want to know how can I pass the persistable package 

My project structure is 

|
|--controller
-------persistable

XXXXX External libraries XXXX
|
|-- Morphia config

**Comment by evanchooly** [comment]
`MorphiaConfig` is immutable.  You need to use the returned reference.
